### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by mmahareeq**
edit a cart page.
- laptop screen :
image

- max- width screen : 500 :
image

- default  : display product a list 
- cart is empty showing a message "The cart is empty, if you want to shop click the button";
image

- The possibility of presenting products in two ways grid or list .

### CODE REVIEW

File: cart-page.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This app is platform where you can sell or buy stuff">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">  
+    <link rel="stylesheet" href="styles/cart-page.css" type="text/css">
+    <title>GITShop</title>
+</head>
+<body>
+    <nav class="nav-page">
```
Comment (bayanseder) [inline cart-page.html]: make the header in this page look like the header in the products page

File: cart-page.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            <li> GITShop</li>
+            <li><a href="index.html" id="return-home">CONTINUE SHOPPING</a></li>
+
+        </ul>
+    </nav>
+    <section class="sec-1">
+    <div class="section-1">
+        <h3 class="h3-pro">Shopping Bag</h3>
+        <span class="span-sh">Prices are subject to change based on the price in effect the day you checkout</span>
+        <table class="tab-pro">
```
Comment (bayanseder) [inline cart-page.html]: we should have 2 way to show result ( list and grid ) ....so we have to do this in js (dom file )

Comment (mmahareeq) [reply cart-page.html]: done

File: scripts/cart-page.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+var list_pro=document.getElementById("list");
```
Comment (bayanseder) [inline scripts/cart-page.js:60]: replace the var with let or const depends on Its kind

**Review by bayanseder — APPROVED** [review]

**Pull request merged**